src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge31.java

```diff
@@ -0,0 +1,92 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+
+import java.nio.charset.StandardCharsets;
+
+import org.owasp.wrongsecrets.RuntimeEnvironment;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Challenge;
+import org.owasp.wrongsecrets.challenges.ChallengeTechnology;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.List;
+import java.util.UUID;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.owasp.wrongsecrets.RuntimeEnvironment.Environment.DOCKER;
+
+@Component
+@Order(31)
+public class Challenge31 extends Challenge {
+
+    public Challenge31(ScoreCard scoreCard) {
+        super(scoreCard);
+    }
+
+    @Override
+    public boolean canRunInCTFMode() {
+        return true;
+    }
+
+
+    @Override
+    public Spoiler spoiler() {
+        return new Spoiler(decode());
+    }
+
+    @Override
+    public boolean answerCorrect(String answer) {
+        return decode().equals(answer);
+    }
+
+    public List<RuntimeEnvironment.Environment> supportedRuntimeEnvironments() {
+        return List.of(DOCKER);
+    }
+
+    @Override
+    public int difficulty() {
+        return 1;
+    }
+
+    @Override
+    public String getTech() {
+        return ChallengeTechnology.Tech.DOCUMENTATION.id;
+    }
+
+    @Override
+    public boolean isLimittedWhenOnlineHosted() {
+        return false;
+    }
+
+    public String decode() {
```

Comment:
```suggestion
    private String getAnswer() {
```
The method is not really about decoding an input, it is about returning the actual answer right :) ?
Next, it should always be private; there is no need to call the method outside of the class.